spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
because you want to require your spec_helper, this is not for when your in the conext of the plugin, but for when you run the test in the conext of logstash-core.